packages/fxa-content-server/server/templates/pages/src/404.html

```diff
@@ -14,6 +14,7 @@
 
     <!-- build:css(.tmp) /styles/main.css -->
     <link rel="stylesheet" href="/styles/main.css" />
+    <link rel="stylesheet" href="/styles/tailwind.out.css" />
```

Comment:
Suggested change to address build error: this matches the stylesheet build for `index.html`. Making this change for all html files in this PR should resolve the error (tested locally and builds successfully).

```suggestion
    <!-- build:css(.tmp) /styles/main.css -->
    <link rel="stylesheet" href="/styles/main.css" />
    <!-- endbuild -->
    <!-- build:css(.tmp) /styles/tailwind.out.css -->
    <link rel="stylesheet" href="/styles/tailwind.out.css" />
    <!-- endbuild -->
```